project/src/main/world/OverworldTouchButtons.tscn

```diff
@@ -7,6 +7,7 @@
 modulate = Color( 1, 1, 1, 0.627451 )
 anchor_right = 1.0
 anchor_bottom = 1.0
+mouse_filter = 2
```

Comment:
This fixes a bug where chat buttons were unclickable; you should mention that in the commit comment.

Reply:
Expanded commit comment; `Fixed bug where chat choices could not be clicked on overworld...`